Examples/SQLServerNetwork.ps1

```diff
@@ -0,0 +1,80 @@
+﻿Configuration SQLNetwork
+{
+    Import-DscResource -Module xSQLServer
+
+    # A Configuration expects at least one Node
+    Node $AllNodes.NodeName
+    {
+        # Set DCM Settings for each Node 
+        LocalConfigurationManager 
+        { 
+            DebugMode = "ForceModuleImport"
```

Comment:
DebugMode should only be enable for debugging.  It's ok to leave it here as a comment and say uncomment to debug.